src/OrbitGl/PageFaultsTrack.cpp

```diff
@@ -174,8 +174,17 @@ void PageFaultsTrack::OnTimer(const orbit_client_protos::TimerInfo& timer_info)
         static_cast<double>(system_page_faults - system_major_page_faults);
     AddValuesAndUpdateAnnotationsForMinorPageFaultsSubtrack(timer_info.start(), values);
   }
+}
+uint32_t PageFaultsTrack::GetNumTimers() const {
```

Comment:
I'm a little bit confused. From the name,  the`GetNumTimers()` method supposes to return the number of timers. For the `PageFaultsTrack`, we extract time series values for the major and minor subtracks from the same page faults timer (See `PageFaultsTrack::OnTimer`) for more details. Hence here we count the number of timers in the `PageFaultsTrack` twice.

Reply:
If we are counting the sum of graph track size here,  maybe we can add comments for the `Track::GetNumTimers()` method to explain that we are collecting different information for the `TimerTrack` and `GraphTrack`.

Reply:
ok, I looked closer and it appears we do not need GetNumTimers outside of ThreadTrack where they are used to sort tracks. I removed this from the base class and from here.